data/data-pipeline/data_pipeline/score/score_narwhal.py

```diff
@@ -1031,10 +1058,15 @@ def add_columns(self) -> pd.DataFrame:
         ]
         self.df[field_names.CATEGORY_COUNT] = self.df[factors].sum(axis=1)
         self.df[field_names.SCORE_N_COMMUNITIES] = self.df[factors].any(axis=1)
+        self._mark_tribal_dacs()
         self.df[
             field_names.SCORE_N_COMMUNITIES
             + field_names.PERCENTILE_FIELD_SUFFIX
         ] = self.df[field_names.SCORE_N_COMMUNITIES].astype(int)
+
         self._mark_donut_hole_tracts()
+        self.df[
```

Comment:
a quick style comment -- why not modify df in place here as per the mark functions?

Reply:
this is a question NOT a suggestion to be clear

Reply:
I think because I noticed a pattern of marking tracts by mutating in place (e.g., with the donut hole calculation) but getting scores and things like that and assigning them (e.g., `self.df[field_names.N_CLIMATE] = self._climate_factor()` etc) so I was following that in this code.